### DISCUSSION TITLE
Export spline as spline

### DISCUSSION THREAD

**Discussion by LKAMinco** [StjerneIdioten/I3D-Blender-Addon#102] [Ideas & Feature Requests]
Possiblity to export spline from blender into GE as spline, would be nice, mainly for mappers, when they are creating roads, and in ge they can shape terrain by spline.

**Comment by StjerneIdioten** [comment]
It actually already exists as the very first issue ever created on the tracker #1 ! 

But I will keep the discussion around if anyone has anything to add ;-)

**Comment by StjerneIdioten** [comment]
This has been implemented in fbec1706c26ea7c5f5e5f15cc596e8bb407b203e and is available in the develop branch
